kitsune/coolsearch/static/coolsearch/js/components/SearchResults.jsx

```diff
@@ -0,0 +1,49 @@
+import SearchResult from './SearchResult.jsx';
+import ResultsStore from '../stores/ResultsStore.js';
+
+
+var getStateFromStores = function () {
+  return {
+    results: ResultsStore.getAll(),
+    count: ResultsStore.getCount()
+  };
+};
+
+export default class SearchResults extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = getStateFromStores();
+    this.state.started = false;
+  }
+
+  componentDidMount() {
+    ResultsStore.addChangeListener(this._onChange.bind(this));
+  }
+
+  _onChange() {
+    this.state.started = true;
+    this.setState(getStateFromStores());
+  }
+
+  render() {
+    var results = this.state.results.map(function (result) {
+      return (
+        <SearchResult
+          key={result.title}
+          data={result}
+        />
+      );
+    });
```

Comment:
ES6 syntax tip: this could be written as

```js
var results = this.state.results.map(result => <SearchResult key={result.title} data={result}/>);
```